README.md

```diff
@@ -52,16 +53,22 @@ All streams will start at 7:30 PM EST
 
 ## Guidelines for Participation
 
-The book club video conference component will only be available to Pittsburgh
-Functional Programming members.  Additionally, this repository will only accept
-PRs from PGH FP members. We would like to keep the discussion as intimate as
-possible for our group. All others are welcome in the Twitch chat to ask
-questions and work on exercises. If you would like to share a code snippet to
-add to the discussion please do so in a gist.
+We are encouraging active participation in the book club from anyone
+interested. To be involved you should tune into the stream during the scheduled
```

Comment:
What is the link that should be put here?  Perhaps it could be:

`[stream](https://www.twitch.tv/chiroptical) during the scheduled`

Reply:
Added.